.github/workflows/quarkus-next.yml

```diff
@@ -0,0 +1,53 @@
+name: Quarkus Next
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+concurrency:
+  # Only cancel jobs for PR updates
+  group: quarkus-next-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  update-quarkus-next-branch:
+    name: Update quarkus-next branch
+    if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.QUARKUS_NEXT_TOKEN }}
```

Comment:
Before merging this, we need to create the `QUARKUS_NEXT_TOKEN` secret.

@Pepo48 Can you please discuss this with @stianst or @abstractj?

Reply:
@vmuzikar @Pepo48 please let me know what you need and we can figure this out.